frappe/database/database.py

```diff
@@ -955,21 +955,17 @@ def exists(self, dt, dn=None, cache=False):
 
 		return self.get_value(dt, dn, ignore=True, cache=cache)
 
-	def count(self, dt, filters=None, debug=False, cache=False):
+	def count(self, dt, filters=None, debug=False, cache=False, distinct:bool = True):
```

Comment:
```suggestion
	def count(self, dt, filters=None, debug=False, cache=False, distinct : bool = True):
```

Likely #16453 is going to be merged before this and we'd have to run Black on it all anyway to resolve the conflicts...